### PR TITLE
Accessability

### PR DESCRIPTION
**Pull request by Kwaku28**
This part of the project required changes to be made to the accessibility of the portfolio.
Changes made to the portfolio include:

1. Page titles - no issues found
2. <a href="https://github.com/touredri/Portfolio/blob/23faedafee81670bbb5a2d064ecf37fff2e249df/index.html#L51-L226">Image text alternative<a/> - alternative texts added for better image description
3. <a href="https://github.com/touredri/Portfolio/blob/23faedafee81670bbb5a2d064ecf37fff2e249df/index.html#L17-L36">Text headings<a/> - headings updated to follow a meaningful hierarchy
4. <a href="https://github.com/touredri/Portfolio/blob/23faedafee81670bbb5a2d064ecf37fff2e249df/style.css#L63">Color contrast<a/> - changes made to list of technologies
5. Resize - no changes done
6. Moving content - no moving content on the portfolio so no changes are done
7. Multimedia - no multimedia content on the portfolio so no changes are done
8. Basic structure of the page - no changes made since it fits the <a href="https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=p3X8dSCaYmSUep5J-0">Figma Design Template 1<a/>

Thank you 🤝

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,10 +14,9 @@
 <body>
     <header>
         <div class="logo">
-          <h4>My logo</h4>
+          <p>My logo</p>
         </div>
```
Comment (amon-cofie) [inline index.html:18]: You have done well in fixing some of the accessibility issues and I will kindly suggest that you fix the color contrast issue I am just realising on your logo 👍  
image

File: index.html [hunk 2]
```diff
@@ -34,7 +33,7 @@ <h1>I’m Drissa, <br> Glad to see you!</h1>
           you see and have a project you need coded, don’t hesitate to contact
           me.
         </p>
-        <h3>Let's connect</h3>
+        <p class="conect">Let's connect</p>
         <ul class="social">
```
Comment (amon-cofie) [inline index.html:37]: Like in the previous comment, you also have color contrast issues on your ```let's connect``` links and I kindly suggest that you fix them 👍 
image
image

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @Kwaku28 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 💪 



## Highlights

🚀 Descriptive pull request
🚀 No linter errors 


## Required Changes ♻️

- [ ] You have done great work on trying to fix the color contrast issues in some parts of your project however, [aditus](https://www.aditus.io/), a great button contrast checker that I use and also recommend 😉, indicates some button contrasts on your page: 
image

- [ ] I also notice a color contrast issue in the contact me section and would kindly urge you to try to fix that 👍 
image




_Check the comments under the review._

## Optional suggestions

[OPTIONAL] I notice your commit messages are not in the imperative form and would advise that going forward you try to form all your commit messages in the imperative form to meet requirements and best practices.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @touredri  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ All colour checks are passing
✔️ All linter checks are passing
✔️ The UI looks good

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by touredri — APPROVED** [review]
perfect

**Pull request merged**